src/polyglot_ffi/parsers/ocaml.py

```
@@ -170,7 +168,7 @@ def _parse_type_definition(
                 return None, lines_consumed
 
         except ParseError as e:
-            raise ParseError(f"Error parsing type definition: {e}", start_line)
+            raise ParseError(f"Error parsing type definition: {e}", line=start_line)
```

Ruff (0.15.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)